ci/pipelines/cf-for-k8s-main.yml

```diff
@@ -352,6 +352,128 @@ jobs:
       params:
         remove: ready-pool
 
+- name: validate-cf-for-k8s-gke-external-blobstore
+  serial: true
+  public: true
+  plan:
+    - in_parallel:
+      - get: cf-for-k8s-develop
+        passed:
+          - test-vendir-sync-on-cf-for-k8s
+          - run-unit-tests
+        trigger: true
+      - get: cf-for-k8s-ci
+
+    - put: ready-pool
+      params: {acquire: true}
+
+    - task: install-minio
+      params:
+        GCP_PROJECT_NAME: ((ci_k8s_gcp_project_name))
+        GCP_PROJECT_ZONE: ((ci_k8s_gcp_project_zone))
+        GCP_SERVICE_ACCOUNT_JSON: ((ci_k8s_gcp_service_account_json))
+      file: cf-for-k8s-ci/ci/tasks/install-minio/task.yml
+      input_mapping:
+        pool-lock: ready-pool
+
+    - task: install-cf
+      input_mapping:
+        pool-lock: ready-pool
+        cf-for-k8s: cf-for-k8s-develop
+      params:
+        GCP_SERVICE_ACCOUNT_JSON: ((ci_k8s_gcp_service_account_json))
+        GCP_PROJECT_NAME: ((ci_k8s_gcp_project_name))
+        GCP_PROJECT_ZONE: ((ci_k8s_gcp_project_zone))
+        APP_REGISTRY_HOSTNAME: https://index.docker.io/v1/
+        APP_REGISTRY_REPOSITORY_PREFIX: ((cf_for_k8s_private_dockerhub.username))
+        APP_REGISTRY_USERNAME: ((cf_for_k8s_private_dockerhub.username))
+        APP_REGISTRY_PASSWORD: ((cf_for_k8s_private_dockerhub.password))
+        USE_EXTERNAL_BLOBSTORE: true
+        DOMAIN: k8s-dev.relint.rocks
+      file: cf-for-k8s-ci/ci/tasks/install-cf-on-gke/task.yml
+
+    - task: run-smoke-test
+      file: cf-for-k8s-ci/ci/tasks/run-smoke-tests/task.yml
+      params:
+        SMOKE_TEST_SKIP_SSL: false
+      input_mapping:
+        cf-for-k8s: cf-for-k8s-develop
+
+    - task: push-test-app
```

Comment:
I think this is not actually required. We had an issue back in June (https://www.pivotaltracker.com/story/show/173280813) where we couldn't delete cf-for-k8s if there were any apps pushed to the platform so we added this as a way to catch that to avoid future degradation. However, I don't think we need to include this test in every CI job.